packaging/setup/plugins/ovirt-engine-setup/ovirt-engine/network/ovirtproviderovn.py

```diff
@@ -1151,6 +1165,47 @@ def _misc_db_entries(self):
         self._add_client_secret_to_db()
         self._set_default_network_provider_in_db()
 
+    @plugin.event(
+        stage=plugin.Stages.STAGE_MISC,
+        after=(
+            oenginecons.Stages.OVN_PROVIDER_OVN_DB,
+        ),
+        condition=lambda self: self._provider_installed,
```

Comment:
Again. Not terribly bad, as I hope it will be harmless, but perhaps better check if we need to update.

Not sure how to best do this, exactly. If you do indeed add a 'KeycloakEnv.CONFIGURED' as discussed in the dwh patch, perhaps you can condition on this - update ovn only if keycloak is ENABLEd but not yet CONFIGURED.

And, to clarify (as I didn't write this in the dwh patch, sorry): I think they should both be in your current KeycloakEnv in engine_common/constants.py and used only from there everywhere - not duplicate ENABLE/D or CONFIGURED in dwh or ovirt-engine-keycloak.

Reply:
That makes sense. Originally I planned to make that unification after those patches/fixes, but I can see the benefits of having it much earlier. I am going to give it a try now when the patches are more or less doing what there are supposed to do.